shared/errutil/errors.go

```diff
@@ -0,0 +1,8 @@
+package errutil
+
+import "errors"
+
+var (
+	// AssignmentNotFoundErr represents the case in which a validator is not currently assigned in an epoch.
+	AssignmentNotFoundErr = errors.New("assignments not found for validator")
```

Comment:
I'm not a fan of this. I think error variables should be in the same package of which they are returned. It is much easier to do

```go
err := assignments.GetAssignments(validator)
if err == assignments.ErrAssignmentsNotFound {
   // Log warning
} else if err != nil {
  // Log error
}
```

Reply:
But we do not have shared visibility between beacon node and validator - where else should I place it? I don't want to duplicate it in both projects.